config/crd/bases/activemonitor.keikoproj.io_healthchecks.yaml

```diff
@@ -127,6 +169,25 @@ spec:
               type: string
             lastSuccessfulWorkflow:
               type: string
+            remedyErrorMessage:
+              type: string
+            remedyFailedCount:
+              type: integer
+            remedyFinishedAt:
+              format: date-time
+              type: string
+            remedyLastFailedAt:
+              format: date-time
+              type: string
+            remedyStatus:
+              type: string
+            remedySuccessCount:
+              type: integer
+            remedyTotal:
```

Comment:
Is `remedyTotal` the total number of times the `remedy` was executed? Can this be changed to `remedyTotalExecutions` or `remedyTotalRuns` or something like that?

Reply:
yes.. remedyTotal is number of times remedy run it includes success and failure of remedy workflow.. sure will update to remedyTotalRuns